taxi/views.py

```diff
@@ -64,7 +88,8 @@
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
+    template_name = "taxi/car_form.html"
     success_url = reverse_lazy("taxi:car-list")
```

Comment:
Ensure that the URL name uses hyphens instead of underscores as per the checklist. It should be `car-list` instead of `car-list`.